client/src/pages/Login/Form/Form.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  TextField,
+  FormControlLabel,
+  Checkbox,
+  Link,
+  Grid,
+  Typography,
+  Box,
+  InputAdornment,
+} from '@material-ui/core';
+import {
+  Formik, Form, Field,
+} from 'formik';
+import * as Yup from 'yup';
+
+import LockIcon from '@material-ui/icons/Lock';
+import EmailIcon from '@material-ui/icons/Email';
+
+import styles from './Form.module.scss';
+
+const initialValues = {
+  email: '',
+  password: '',
+
+};
+
+const validationSchema = () => Yup.object().shape({
+  email: Yup.string()
+    .email('Invalid email.')
+    .required('This field is required'),
+  password: Yup.string()
+    .min(6, 'Password must have at least 8 characters.')
+    .required('This field is required'),
+});
+
+const LoginForm = () => (
```

Comment:
For now the login form is not necessary, but the same structure can help us to build the create event form. Hence, this block code should be commented like a dummy code (instead of remove the installed dependencies and formik config)